packages/portalnetwork/test/integration/eth/getBlockByNumber.spec.ts

```diff
@@ -120,101 +53,162 @@ describe('eth_getBlockByNumber', () => {
       },
     })
 
+    const node2 = await PortalNetwork.create({
+      transport: TransportLayer.NODE,
+      supportedNetworks: [{ networkId: NetworkId.HistoryNetwork }],
+      config: {
+        enr: enr2,
+        bindAddrs: {
+          ip4: initMa2,
+        },
+        peerId: id2,
+      },
+    })
+
+    // node1.enableLog('*Portal*,-uTP*')
+    // node2.enableLog('*Portal*,-uTP*')
+
     await node1.start()
+    await node2.start()
+    const network1 = node1.networks.get(NetworkId.HistoryNetwork) as HistoryNetwork
+    const network2 = node2.networks.get(NetworkId.HistoryNetwork) as HistoryNetwork
+
+    const epochData = require('../../testData/testEpoch.json')
+    const block1000 = require('../../testData/testBlock1000.json')
+    const epoch = epochData.serialized
+
+    const blockRlp = block1000.raw
+    const blockHash = block1000.hash
+    const proof = await generatePreMergeHeaderProof(1000n, fromHexString(epoch))
+    await addRLPSerializedBlock(blockRlp, blockHash, network1, proof)
+    await network1.sendPing(network2?.enr!.toENR())
+    const retrieved = await node2.ETH.getBlockByNumber(1000, false)
+    it('should retrieve header', () => {
+      assert.equal(Number(retrieved!.header.number), 1000, 'retrieved expected header')
+    })
 
-    const beacon = node1.networks.get(NetworkId.BeaconChainNetwork) as BeaconLightClientNetwork
+    await node1.stop()
+    await node2.stop()
+  },
+  { timeout: 30000 },
+)
+describe('should find a block using "latest" and "finalized"', async () => {
```

Comment:
We need there to be `it('should...)` inside of these describe statements or we're going to have async weirdness.  `describe` should never be async as far as I remember from previous struggles with async stuff in `vitest`.

Reply:
I think is is the other way around (async `it()` inside of non-async `describe`) that causes issues.

This does have an `it` test, at the bottom, that checks the results after all the async stuff happens inside of `describe`

Reply:
Oh, really?  Then maybe I've got things backwards.  Will take a look at vitest here.